Makefile

```diff
@@ -195,7 +195,7 @@ test: all
 	$(MAKE) build-addons
 	$(MAKE) build-addons-napi
 	$(MAKE) cctest
-	$(PYTHON) tools/test.py --mode=release -J \
+	$(PYTHON) tools/test.py --mode=release --flaky-tests=$(FLAKY_TESTS) -J \
```

Comment:
This change LGTM. 

//cc: @joaocgreis  in case he has any objection. To give you context, this was to not see flaky test result in our buddy testing on linux.